UI/NativeApp.cpp

```diff
@@ -335,11 +338,19 @@ static void PostLoadConfig() {
 		i18nrepo.LoadIni(g_Config.sLanguageIni, langOverridePath);
 }
 
-void CreateDirectoriesAndroid() {
+static bool CreateDirectoriesAndroid() {
 	// On Android, create a PSP directory tree in the external_dir,
 	// to hopefully reduce confusion a bit.
-	INFO_LOG(IO, "Creating %s", (g_Config.memStickDirectory / "PSP").c_str());
-	File::CreateFullPath(g_Config.memStickDirectory / "PSP");
+
+	Path pspDir = g_Config.memStickDirectory / "PSP";
+
+	INFO_LOG(IO, "Creating '%s' and subdirs:", pspDir.c_str());
+	File::CreateFullPath(pspDir);
```

Comment:
Should this check if it ends in PSP already?

-[Unknown]

Reply:
Done